contracts/governance/Governor.sol

```diff
@@ -20,73 +25,39 @@ abstract contract Governor is IGovernor, EIP712, Context {
         bool canceled;
     }
 
-    string public override name;
-    string public override version;
+    string private _name;
+    string private _version;
 
     mapping (uint256 => Proposal) private _proposals;
     mapping (uint256 => mapping (address => bool)) private _votes;
 
+    /**
+     * @dev Sets the value for {name} and {version}
+     */
     constructor(string memory name_, string memory version_)
     EIP712(name_, version_)
     {
-        name = name_;
-        version = version_;
+        _name = name_;
```

Comment:
Isn't it better to use `_name = name` or `name = _name`? (to avoid the amount of underscores)

Reply:
- `name` is the external viewing function (part of the interface)
- `_name` is the storage slot storing it (private to avoid inheriting contract to accidentally write to it)
- `name_` is the constructor parameter.

All tree have different names to avoid the collision warning.